### PR TITLE
feat: polling order book data every 5 sec

### PR DESCRIPTION
**Pull request by coldsnap666**

### CODE REVIEW

File: src/pages/Trade/components/TradePriceTable/index.tsx [hunk 1]
```diff
@@ -190,30 +190,52 @@ const TradePriceTable = () => {
 
   const [currentPrice, setCurrentPrice] = useState('--')
 
+  const getPlaceTableTimer = useRef<ReturnType<typeof setInterval> | undefined>()
```
Comment (homu-chain) [inline src/pages/Trade/components/TradePriceTable/index.tsx]: Is the `ref` necessary here? A ref dep by `useEffect(..., [ref])` is awalys the same object

Maybe this is more simple?
```tsx

useEffect(() => {
  function getPriceTable() {
  }
  getPriceTable()

  let intervalId = setInterval(getPriceTable, 5000)

  return () => {
    clearInterval(intervalId)
  }
}, [sudt, lockHash])

```

Comment (coldsnap666) [reply src/pages/Trade/components/TradePriceTable/index.tsx]: Yes, every time the trade pair(sudt) changes, we need to clear the previous interval.

Comment (homu-chain) [reply src/pages/Trade/components/TradePriceTable/index.tsx]: when the sudt has changed, the `useEffect` will be triggered, the`interval` would be clear like this

`componentDidMount -> useEffect(getPriceTable) -> sudt changed -> exec useEffect exit function(clearInterval) -> useEffect(getPriceTable)`

We can try the simple [demo](https://codesandbox.io/s/late-water-7f3s8?file=/src/App.js)

Comment (yuche) [reply src/pages/Trade/components/TradePriceTable/index.tsx]: Good to know.

Comment (coldsnap666) [reply src/pages/Trade/components/TradePriceTable/index.tsx]: Good to know.

**Review by homu-chain — APPROVED** [review]

**Pull request merged**